arm1:
  a1:
  - 0.8226810097694397
  - -0.014591719023883343
  - 0.2020411193370819
  - 0.046124618500471115
  - 1.3696004152297974
  - 0.7891924381256104
  a2:
  - 0.7096562385559082
  - 0.11837524175643921
  - 0.06316603720188141
  - 0.016427431255578995
  - 1.350975751876831
  - 0.6954066157341003
  a3:
  - 0.6030158400535583
  - 0.3259360194206238
  - -0.20892660319805145
  - 0.01725444197654724
  - 1.4485507011413574
  - 0.6317681074142456
  a4:
  - 0.5197446346282959
  - 0.5556820034980774
  - -0.5667150616645813
  - 0.02684650756418705
  - 1.543516755104065
  - 0.5124005079269409
  b1:
  - 0.6521790623664856
  - -0.19495514035224915
  - 0.3878929018974304
  - 0.04703934118151665
  - 1.3201452493667603
  - 0.6473386883735657
  b2:
  - 0.565054714679718
  - 0.007480683736503124
  - 0.19409053027629852
  - 0.015807172283530235
  - 1.3743937015533447
  - 0.5480865836143494
  b3:
  - 0.45699837803840637
  - 0.20531782507896423
  - -0.05921894684433937
  - 0.017285767942667007
  - 1.3691760301589966
  - 0.3896683156490326
  b4:
  - 0.3912886381149292
  - 0.44841501116752625
  - -0.4009872078895569
  - 0.026852771639823914
  - 1.537522554397583
  - 0.36349648237228394
  c1:
  - 0.3972907066345215
  - -0.292811781167984
  - 0.48709025979042053
  - 0.05549740046262741
  - 1.3628815412521362
  - 0.38409852981567383
  c2:
  - 0.36488693952560425
  - -0.10852629691362381
  - 0.2932816445827484
  - 0.01579464226961136
  - 1.3459131717681885
  - 0.33593738079071045
  c3:
  - 0.28966033458709717
  - 0.12437733262777328
  - 0.035617828369140625
  - 0.017279502004384995
  - 1.3770544528961182
  - 0.2293451428413391
  c4:
  - 0.24443165957927704
  - 0.37890854477882385
  - -0.2846042811870575
  - 0.027498090639710426
  - 1.4727658033370972
  - 0.22915880382061005
  d1:
  - 0.10134634375572205
  - -0.3821852505207062
  - 0.561020016670227
  - 0.06389907002449036
  - 1.3807814121246338
  - 0.07574094086885452
  d2:
  - 0.12023600935935974
  - -0.13799166679382324
  - 0.3444623053073883
  - 0.01575705222785473
  - 1.346886396408081
  - 0.061971746385097504
  d3:
  - 0.11237942427396774
  - 0.07431815564632416
  - 0.10803761333227158
  - 0.017530111595988274
  - 1.3629850149154663
  - 0.046194110065698624
  d4:
  - 0.08421094715595245
  - 0.3607080578804016
  - -0.272430956363678
  - 0.025850336998701096
  - 1.4914734363555908
  - 0.06618533283472061
  e1:
  - -0.1746620535850525
  - -0.3643982708454132
  - 0.5489907264709473
  - 0.047383926808834076
  - 1.3928837776184082
  - -0.1864328682422638
  e2:
  - -0.09333939105272293
  - -0.14732061326503754
  - 0.3562597334384918
  - 0.01575705222785473
  - 1.3469175100326538
  - -0.1296064555644989
  e3:
  - -0.07697460800409317
  - 0.0988653153181076
  - 0.06589768081903458
  - 0.00939157884567976
  - 1.4111461639404297
  - -0.0828947126865387
  e4:
  - -0.06767074018716812
  - 0.3737773299217224
  - -0.28743618726730347
  - 0.025812745094299316
  - 1.5147879123687744
  - -0.09453123807907104
  f1:
  - -0.45086470246315
  - -0.26929208636283875
  - 0.4555322825908661
  - 0.033506445586681366
  - 1.3867549896240234
  - -0.45918723940849304
  f2:
  - -0.3125848174095154
  - -0.08416709303855896
  - 0.2976923882961273
  - 0.01576331816613674
  - 1.3471038341522217
  - -0.3913765251636505
  f3:
  - -0.28203555941581726
  - 0.12658269703388214
  - 0.05963245406746864
  - 0.03393874689936638
  - 1.3879766464233398
  - -0.3165259063243866
  f4:
  - -0.22941391170024872
  - 0.4057299792766571
  - -0.33800286054611206
  - 0.02977236919105053
  - 1.525606393814087
  - -0.24908789992332458
  g1:
  - -0.6642834544181824
  - -0.18797567486763
  - 0.3885068893432617
  - 0.019509924575686455
  - 1.3486567735671997
  - -0.7262682914733887
  g2:
  - -0.5345305800437927
  - 0.03235364332795143
  - 0.16782043874263763
  - 0.01800626888871193
  - 1.3844980001449585
  - -0.5889282822608948
  g3:
  - -0.44609686732292175
  - 0.24830356240272522
  - -0.09659730643033981
  - 0.03144518658518791
  - 1.4591519832611084
  - -0.4509050250053406
  g4:
  - -0.37014973163604736
  - 0.4728744626045227
  - -0.4375072419643402
  - 0.029672125354409218
  - 1.5388270616531372
  - -0.3986855745315552
  h1:
  - -0.8028954267501831
  - -0.028394019231200218
  - 0.23733940720558167
  - 0.007900454103946686
  - 1.365676760673523
  - -0.8208513259887695
  h2:
  - -0.6778852939605713
  - 0.18880268931388855
  - -0.0272725448012352
  - 0.017818311229348183
  - 1.43012273311615
  - -0.7132237553596497
  h3:
  - -0.5930352807044983
  - 0.3326836824417114
  - -0.21970906853675842
  - 0.03008563257753849
  - 1.4660263061523438
  - -0.6187649965286255
  h4:
  - -0.5088368654251099
  - 0.5616340041160583
  - -0.5507888197898865
  - 0.029665861278772354
  - 1.5543664693832397
  - -0.5095742344856262
  not_playing_home:
  - 0.0005826663109473884
  - -1.447343111038208
  - 0.20036830008029938
  - -0.0016352247912436724
  - 1.6672946214675903
  - 0.010715128853917122
  playing_home:
  - 0.0005325444508343935
  - -0.056863222271203995
  - -0.5482138395309448
  - -0.0016101639484986663
  - 1.4187554121017456
  - 0.010715128853917122
arm1_2:
  not_playing_home:
  - 0.0005826663109473884
  - -1.447343111038208
  - 0.20036830008029938
  - -0.0016352247912436724
  - 1.6672946214675903
  - 0.010715128853917122
  - 0.0005826663109473884
  - -1.447343111038208
  - 0.20036830008029938
  - -0.0016352247912436724
  - 1.6672946214675903
  - 0.010715128853917122
  playing_home:
  - 0.0005325444508343935
  - -0.056863222271203995
  - -0.5482138395309448
  - -0.0016101639484986663
  - 1.4187554121017456
  - 0.010715128853917122
  - 0.0005325444508343935
  - -0.056863222271203995
  - -0.5482138395309448
  - -0.0016101639484986663
  - 1.4187554121017456
  - 0.010715128853917122
arm2: {}
gripper:
  grab_piece: 
  - 1.0563769340515137
  - 1.0563769340515137
  ungrab_piece:
  - 0.7712011933326721
  - 0.7712011933326721